applications/FluidDynamicsApplication/python_scripts/check_and_prepare_model_process_fluid.py

```diff
@@ -62,7 +62,12 @@ def Execute(self):
 
         fluid_computational_model_part.AddConditions(list(list_of_ids))
 
-        #verify the orientation of the skin
+        #verify the orientation of the skin (only implemented for tris and tets)
+        geometry = self.main_model_part.GetElement(1).GetGeometry()
+        is_simplex = geometry.LocalSpaceDimension() + 1 == geometry.PointsNumber()
+        if not is_simplex:
```

Comment:
I'd throw a warning in here saying that `TetrahedralMeshOrientationCheck` will not be executed.